packages/compass-components/src/components/file-picker-dialog.tsx

```diff
@@ -311,7 +311,16 @@ export function createElectronFileInputBackend<ElectronWindow>(
   };
 }
 
-function FileInput({
+/**
+ * This component is not intended to work in a browser environment. It is designed
+ * to be used in environments like Electron where you have access to nodes fs module
+ * to read/write files.
+ *
+ * For browser environments, you should use the `FileSelector` component instead.
```

Comment:
I just really want to make this point clear, browser environment shouldn't be an exception, it should be a default

```suggestion
 * Always use `FileSelector` component, only use `FilePickerDialog` if you absolutely know what you're doing
```